rust/ql/lib/codeql/rust/controlflow/internal/ControlFlowGraphImpl.qll

```diff
@@ -461,6 +472,18 @@ class ParenExprTree extends StandardPostOrderTree, ParenExpr {
 // This covers all patterns as they all extend `Pat`
 class PatExprTree extends LeafTree instanceof Pat { }
 
+class PathTree extends StandardPostOrderTree, Path {
```

Comment:
Shouldn't `PathTree`s be simply a `LeaveTree`? I'm not sure any evaluation takes place inside a `Path`.